lib/commons/dom/is-skip-link.js

```diff
@@ -0,0 +1,36 @@
+/* global dom */
+
+// test for hrefs that start with # or /# (for angular)
+const isInternalLinkRegex = /^\/?#[^/!]/;
+
+/**
+ * Determines if element is a skip link
+ * @method isSkipLink
+ * @memberof axe.commons.dom
+ * @instance
+ * @param  {Element} element
+ * @return {Boolean}
+ */
+dom.isSkipLink = function(element) {
```

Comment:
We'll need to have tests for this. Feel free to move some of the tests of the check into a separate tests for this commons.